### PR TITLE
temp vars with lambda to make msvs happy

### PR DESCRIPTION
**Pull request by fprimex**
Redoing the edits to make MSVS work.

### CODE REVIEW

File: mainwindow.cpp [hunk 1]
```diff
@@ -40,9 +40,10 @@ MainWindow::MainWindow(QWidget *parent) :
         my_slots[i]->setText("Slot " + QString::number(i));
         my_slots[i]->setActionGroup(my_slots_group);
         SaveSlot->addAction(my_slots[i]);
-        connect(my_slots[i], &QAction::triggered,[=](bool checked){
+        QAction *temp_slot = my_slots[i];
+        connect(temp_slot[i], &QAction::triggered,[=](bool checked){
             if (checked) {
-                int slot = my_slots[i]->text().remove("Slot ").toInt();
+                int slot = temp_slot[i]->text().remove("Slot ").toInt();
```
Comment (loganmc10) [inline mainwindow.cpp]: this should be temp_slot, not temp_slot[i]

File: mainwindow.cpp [hunk 2]
```diff
@@ -40,9 +40,10 @@ MainWindow::MainWindow(QWidget *parent) :
         my_slots[i]->setText("Slot " + QString::number(i));
         my_slots[i]->setActionGroup(my_slots_group);
         SaveSlot->addAction(my_slots[i]);
-        connect(my_slots[i], &QAction::triggered,[=](bool checked){
+        QAction *temp_slot = my_slots[i];
+        connect(temp_slot[i], &QAction::triggered,[=](bool checked){
```
Comment (loganmc10) [inline mainwindow.cpp]: this should be temp_slot, not temp_slot[i]

File: mainwindow.cpp [hunk 3]
```diff
@@ -160,8 +161,9 @@ void MainWindow::updateOpenRecent()
         recent[i] = new QAction(this);
         recent[i]->setText(list.at(i));
         OpenRecent->addAction(recent[i]);
+        QAction *temp_recent = recent[i];
         connect(recent[i], &QAction::triggered,[=](){
```
Comment (loganmc10) [inline mainwindow.cpp]: this should be temp_recent, not recent[i]

**Comment by fprimex** [comment]
Updated

**Comment by loganmc10** [comment]
There is still one issue, it has:
```
connect(recent[i], &QAc.....
```
but should be:
```
connect(temp_recent, &QAc......
```

**Comment by fprimex** [comment]
Sorry, just trying to push this through and busy with 1000 other things. I got my multi-machine dev environment for Qt going again today, so shortly I'll be able to quickly pull down and test this stuff everywhere.

**Comment by fprimex** [comment]
This one builds and loads :)

**Comment by loganmc10** [comment]
No problem, thanks for the help!

**Pull request merged**